extensions/short-io/README.md

```diff
@@ -0,0 +1,15 @@
+# Short.io
+
+Shorten links using your own domain via Short.io.
+
+**Tips**:
+
+- The first time you use 𝐒𝐡𝐨𝐫𝐭𝐞𝐧 𝐋𝐢𝐧𝐤 and 𝐒𝐞𝐚𝐫𝐜𝐡 𝐋𝐢𝐧𝐤𝐬 commands, they will use the first domain of your domains list.
+- You can set default domain for 𝐒𝐡𝐨𝐫𝐭𝐞𝐧 𝐋𝐢𝐧𝐤 and 𝐒𝐞𝐚𝐫𝐜𝐡 𝐋𝐢𝐧𝐤𝐬 commands via the 𝐒𝐡𝐨𝐫𝐭𝐞𝐧 𝐋𝐢𝐧𝐤 𝐖𝐢𝐭𝐡 𝐃𝐨𝐦𝐚𝐢𝐧 command (action: Set Default Domain, shortcut: Cmd+Enter).
+
+**Get your secret API key [here](https://app.short.io/settings/integrations/api-key)**:
```

Comment:
I tried with a public API key and got a 401, probably worth pointing out that users should create a private API key

Reply:
I haven't tested the public key yet, I'll test it first, and if there is indeed a problem, I'll modify the README to tell the user to request the private key

Reply:
I added tips to the README: 
- Do not turn on the _Public key_ switch, this extension does not support public key.